content/tutorials/publish-subscribe.textile

```diff
@@ -98,17 +98,17 @@ blang[android].
   And then run the demo on your Android device. Check "Android Developers website":https://developer.android.com/training/basics/firstapp/running-app.html if you are not familiar on how to run an Android Project. Don't forget to replace your @ExampleActivity#API_KEY@ field with "Ably API key":https://faqs.ably.com/setting-up-and-managing-api-keys.
 
 blang[javascript].
-  The complete source code for each step of "this tutorial is available on Github":https://github.com/ably/tutorials/commits/publish-subscribe-javascript.
-
-  We recommend that you clone the repo locally:
+  The solution code for this tutorial is "available on Github":https://github.com/ably/js-publish-subscribe-tutorial.git.
 
+  To run the demo, perform the following tasks:
+ 
+  #1 Clone the repo locally:
```

Comment:
This looks like it is expecting to be a numbered list, but in the web view `#` will not display as a numbered list. If you want the steps to be numbered you'll have to actually number them, rather than use `#`.